helm/charts/efs-provisioner/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Is there any license related injecting their exact helm files ?

Reply:
No, I searched around in their official repo and I can't seem to find any license terms or agreemenst